flathub.json

```diff
@@ -1,3 +1,5 @@
 {
-    "skip-icons-check": true
+    "skip-icons-check": "true",
+    "end-of-life": "The application has been renamed to io.sourceforge.atanks.",
+    "end-of-life-rebase": "io.sourceforge.atanks"
```

Comment:
There is no project with this ID.

Reply:
see https://github.com/flathub/flathub/pull/3449